src/test/testrunner.ts

```diff
@@ -675,10 +678,10 @@ export class JDServiceTestRunner extends JDServiceClient {
                     )
                     const register = service.register(pkt.identifier)
                     this.registers[regName] = register
-                    this.environment[regName] = register.intValue
+                    this.environment[regName] = register.unpackedValue ? register.unpackedValue[0] : register.intValue
```

Comment:
you should not have to use register.intValue, this property should be deprecated. Use
```
register.unpackedValue?.[0]
```